### ISSUE TITLE
add IRCClient.back()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3366]
|<img alt="benjamin's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| benjamin reported|
|-|-|
|Trac ID|trac#3366|
|Type|enhancement|
|Created|2008-07-23 23:44:21Z|

This simple patch adds a back() (converse of away()) method to IRCClient for convenience.

Attachments:

* [irc_back.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/11f/11ff2346b384b19190353d5d04521327b7486e6e/7e8af3d8ef03ee7aba9994d815c1b144eee01746.patch) (589 bytes) - added by benjamin on 2008-07-23 23:44:44Z - 
* [irc_back2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/11f/11ff2346b384b19190353d5d04521327b7486e6e/a1f6d1fc1c3fcdcfcbbd0c1fbb791d507626a4c4.patch) (1671 bytes) - added by benjamin on 2008-07-24 12:53:40Z - 
* [irc-back.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/11f/11ff2346b384b19190353d5d04521327b7486e6e/908b30b14fdd88d7581c1c9b520df7eeb9a26a6a.patch) (2622 bytes) - added by exarkun on 2008-07-24 14:49:09Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3366 3366
type__enhancement enhancement
reporter__benjamin benjamin
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__words words
keywords__ 
time__1216856661000000 1216856661000000
changetime__1218943673000000 1218943673000000
version__None None
owner__ 
cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to benjamin|
|-|-|

Can you write a unit test for this?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

Thanks!

I tweaked the docstrings slightly and made similar changes to the other tests in that `TestCase` and I adjusted the whitespace in the docstring for the implementation and gave the test module a docstring.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_4"></a>|@radix set owner to @exarkun|
|-|-|

1. ```message = 'Sorry, I\'m not here.' ```

Python supports double quote strings btw

Otherwise, +1, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [24606]) Apply irc-back.patch

Author: benjamin, exarkun
Reviewer: radix
Fixes: [#3366](https://github.com/twisted/twisted/issues/3366)

Add a `back` method to mirror the `away` method on `IRCClient`.  Also
add a test for the `away` method.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
